src/model/page.urls.ts

```diff
@@ -13,10 +13,10 @@ export const ENTER_INFORMATION: string = SEPARATOR + Templates.ENTER_INFORMATION
 export const DOCUMENT_UPLOAD: string = SEPARATOR + Templates.DOCUMENT_UPLOAD;
 export const DOCUMENT_UPLOAD_CONTINUE: string = SEPARATOR + "document-upload-continue"; // not a template
 export const REMOVE_DOCUMENT: string = SEPARATOR + Templates.REMOVE_DOCUMENT + "/:attachmentId";
-export const DOWNLOAD_ATTACHMENT: string =
+export const DOCUMENT_DOWNLOAD: string =
```

Comment:
I can see why you changed this to standardise on the terminology but we do still refer to attachments in a number of places, so still have a mixture of terms going on. Probably in the API too.

Reply:
i know, just wanted the controllers aligned, kept it as attachment in function names as that's what it is called in the API. Without changing the api , i tried to get it as close as I could.